tox.ini

```diff
@@ -25,4 +25,4 @@ deps =
 commands =
     flake8 channels tests
     black --check channels tests
-    isort --check-only --diff channels tests
+    isort --check --diff channels tests
```

Comment:
Is `--check-only` deprecated or something? Or you just think it looks prettier? 😄

Reply:
Yeah, I just thought it was "prettier" because `isort` is often times used along side `black`, and this way the commands are the same.